dates.py

```diff
@@ -1,10 +1,59 @@
 import re
+import logging
+
 from google.cloud import storage
+from google.auth.exceptions import DefaultCredentialsError
+
+# Create/configure stream handler
+sh = logging.StreamHandler()
+fmt = logging.Formatter('%(name)s.py - %(levelname)s - %(message)s')
+sh.setFormatter(fmt)
+
+# Register stream handler
+logger = logging.getLogger(__name__)
+logger.addHandler(sh)
```

Comment:
This logging boilerplate looks like it would be really useful for other files. Note to self that this would be useful in its own util. (no changes needed in this PR)